### ISSUE TITLE
Add clarifications to README setup instructions

### ISSUE THREAD

**Issue by nrs1729** [CircuitVerse/CircuitVerse#733]
**Is your feature request related to a problem? Please describe.**
Some of the instructions are unclear and incomplete, making them hard to follow. 

**Describe the solution you'd like**
Add instructions (or a link to instructions) for installing and running redis server and for setting up postgres, and rephrase the line about configuring the database.

**Are you working on this? (Yes/No)**
Yes

**Comment by sal2701** [comment]
Hey @nrs1729, this can be a GCI task, I'll add one so you can claim it then

**Comment by nrs1729** [comment]
Thanks!

**Comment by satu0king** [comment]
These are the instructions if the developer chooses not to use Docker correct ?

**Comment by satu0king** [comment]
These are the instructions if the developer chooses not to use Docker correct ?

**Comment by nrs1729** [comment]
Yeah without Docker

**Comment by nrs1729** [comment]
I've already done two beginner tasks so someone else can claim it if they want.

**Comment by sal2701** [comment]
@nrs1729 That's very kind of you. You can send a PR, in-case no one claims this within a few days we'll merge yours because this is a very important part of the ReadMe and we need it to be done as soon as possible.

**Comment by nrs1729** [comment]
Okay cool, I made a PR last week for a task adding WSL instructions (https://github.com/CircuitVerse/CircuitVerse/pull/720) so is it okay if I use that as a base instead of the currently merged README (create a branch of the branch I made for the task)? I know some other people edited the README for docker instructions etc.

**Comment by nrs1729** [comment]
No one has claimed it so I had a go and made a PR https://github.com/CircuitVerse/CircuitVerse/pull/749 (based of the currently merged readme - the instructions I/other people added for WSL and docker are in different PRs), is that alright?
